modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -0,0 +1,85 @@
+
+== Before you Begin
+
+Before installing Anypoint Runtime Fabric in a self-managed Kubernetes service, ensure the following:
+
+* You have installed and configured your Kubernetes cluster as follows:
++
+- Running Kubernetes version 1.14.x to 1.18.x.
+- Running an ingress controller to send external requests to deployments.
+* You have downloaded and installed the `rtfctl` command-line utility.
++
+`rtfctl` is supported on Windows, MacOS (Darwin), and Linux. Download this utility using the URLs below:
++
+*Windows:*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-windows/latest -o rtfctl
+----
++
+*MacOS (Darwin):*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-darwin/latest -o rtfctl
+----
++
+*Linux:*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
+----
+
+== Create a Runtime Fabric using Runtime Manager
+
+To install Runtime Fabric on a self-managed Kubernetes cluster, first create a Runtime Fabric using Runtime Manager. This is required to obtain the activation data which is needed during installation.
+
+. From Anypoint Platform, select Runtime Manager.
+. Click *Runtime Fabrics*.
+. Click *Create Runtime Fabric*.
+. Enter the name of the new Runtime Fabric, then select one of the following options:
++
+* Amazon Elastic Kubernetes Service
+* Azure Kubernetes Service
+
+. Click *Next*.
+. Review the *Support responsibility* disclaimer, then if you agree click *Continue*.
++
+Runtime Manager creates the Runtime Fabric and displays the Activation State page. This page displays the activation data used to install Runtime Fabric on a Kubernetes service. Copy this data to the clipboard for use in the next section. 
+
+
+== Install Runtime Fabric on a Self-Managed Kubernetes Cluster
+
+After creating a Runtime Fabric and getting the activation data, install Runtime Fabric into your Kubernetes service using the `rtfctl` command.
+
+[NOTE]
+====
+Run `rtfctl --help` to see a list of options for this command.
+====
+
+. From the command line, run `rtfctl --validate`.
++
+The `--validate` option verifies that the Kubernetes cluster is running, that all required components exist, and that all required services are available. The `rtfctl` command outputs any incompatibilities with Kubernetes cluster.
+
+. Run the `rtfctl --install` command using the following syntax:
++
+----
+rtfctl --install <activation_data>
+----
++
+Where <activation_data> is the activation data obtained after creating the Runtime Fabric using Runtime Manager.
++
+During installation, the `rtfctl` command displays any errors.
+
+. After the installation has completed succesfully, update the Mule license:
++
+----
+rtfctl --update mule-license
+----
+
+== Validating Runtime Fabric on a Self-Managed Kubernetes Cluster
+
+*SL Note: How would the user validate that RTF is running? Deploy an app and hit the URL?*
```

Comment:
There are multiple ways to validate:
1. Once rtfctl install command is finished, it outputs that Runtims Fabric is ready
2. User can go back to the Runtime Manager and confirm if the newly created Runtime Fabric is shown as `active`
3. User can also run `rtfctl status` command to see if everything looks good.

Reply:
I think option 1 is good to point out in the prior section.

Option 2 is the right segway to ask the user to complete their configuration by associating at least one environment to their RTF, and to configure the ingress details.

Reply:
Added option 1 to the end of the procedure.

Described option 2 in this section. I will follow up with team to add more detail.